lib/etc/openturns.conf.in

```diff
@@ -379,6 +379,7 @@
   <Sample-CommentMarkers         value_str="#"    />
   <Sample-PrintEllipsisSize      value_int="3"    />
   <Sample-PrintEllipsisThreshold value_int="1000" />
+  <Sample-PrintFormat            value_str="%.7g" />
```

Comment:
Ok. I will fix this. Notice, however, that, if I wrote the C++ markdown repr, it would not be possible to use the same format in the C++ and the Python layer.

Reply:
Fixed at 7eb9348df